aws/data_source_aws_signer_signing_job_test.go

```diff
@@ -0,0 +1,85 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceAWSSignerSigningJob_basic(t *testing.T) {
+	rString := acctest.RandString(48)
+	profileName := fmt.Sprintf("tf_acc_sp_basic_%s", rString)
+	dataSourceName := "data.aws_signer_signing_job.test"
+	resourceName := "aws_signer_signing_job.job_test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
```

Comment:
Followup note: Functionality will not be launching in GovCloud (US) immediately, will need to skip testing there for now:

```
        Error: error creating Signer signing profile: ResourceNotFoundException: Signing platform not found for ID AWSLambda-SHA384-ECDSA
        {
          RespMetadata: {
            StatusCode: 404,
            RequestID: "3fbe82f2-4fc0-4283-9862-d183964eec1a"
          },
          Code_: "PlatformNotFound",
          Message_: "Signing platform not found for ID AWSLambda-SHA384-ECDSA"
        }
```